worlds/alttp/Rom.py

```diff
@@ -1723,7 +1723,7 @@ def write_custom_shops(rom, world, player):
             price_data = get_price_data(item['price'], item["price_type"])
             replacement_price_data = get_price_data(item['replacement_price'], item['replacement_price_type'])
             slot = 0 if shop.type == ShopType.TakeAny else index
-            if not item['item'] in item_table:  # item not native to ALTTP
+            if not item['item'] in item_table or not item.game == "A Link to the Past":  # item not native to ALTTP
```

Comment:
item here is a custom dict for shops and does not contain an Item class object, therefore item.game will be an attributerror